Gemfile

```diff
@@ -1,31 +1,38 @@
 source 'http://rubygems.org'
 
-# Specify your gem's dependencies in buildingsync.gemspec
 gemspec
 
-if File.exist?('../OpenStudio-extension-gem')
-  # gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
-  gem 'openstudio-extension', path: '../OpenStudio-extension-gem'
-else
-  gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
-end
+# Local gems are useful when developing and integrating the various dependencies.
+# To favor the use of local gems, set the following environment variable:
+#   Mac: export FAVOR_LOCAL_GEMS=1
+#   Windows: set FAVOR_LOCAL_GEMS=1
+# Note that if allow_local is true, but the gem is not found locally, then it will
+# checkout the latest version (develop) from github.
+allow_local = ENV['FAVOR_LOCAL_GEMS']
+
+# Uncomment the extension and common measures gem if you need to test local development versions. Otherwise
+# these are included in the model articulation gem.
+#
+# if allow_local && File.exist?('../OpenStudio-extension-gem')
+#   gem 'openstudio-extension', path: '../OpenStudio-extension-gem'
+# elsif allow_local
+#   gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
+# end
+#
+# if allow_local && File.exist?('../openstudio-common-measures-gem')
+#   gem 'openstudio-common-measures', path: '../openstudio-common-measures-gem'
+# elsif allow_local
+#   gem 'openstudio-common-measures', github: 'NREL/openstudio-common-measures-gem', branch: 'develop'
+# end
 
-if File.exist?('../openstudio-model-articulation-gem')
-  # gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'develop'
+if allow_local && File.exist?('../openstudio-model-articulation-gem')
   gem 'openstudio-model-articulation', path: '../openstudio-model-articulation-gem'
-else
+elsif allow_local
   gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'DA'
 end
 
-if File.exist?('../openstudio-common-measures-gem')
-  # gem 'openstudio-model-articulation', github: 'NREL/openstudio-common-measures-gem', branch: 'develop'
-  gem 'openstudio-common-measures', path: '../openstudio-common-measures-gem'
-else
-  gem 'openstudio-common-measures', github: 'NREL/openstudio-common-measures-gem', branch: 'develop'
-end
-
-# OpenStudio Measure Tester includes the dependencies for running unit tests, coverage, and rubocop
-gem 'openstudio_measure_tester', '0.1.7'
-
 # simplecov has an unneccesary dependency on native json gem, use fork that does not require this
 gem 'simplecov', github: 'NREL/simplecov'
```

Comment:
gem 'simplecov' ... included in Gemfile of openstudio-extension-gem...also remove from here then?

Reply:
Should we also clean up the Gemfile for [EDV](https://github.com/NREL/edv-experiment-1/blob/a1d8bb2f68fe787c04913d2175100767ba000f5e/Gemfile#L13-L32)?  Or this comes later?

Reply:
yeah, we need to remove this too. 

We can cleanup EDV whenever. Does EDV use the openstudio-model-articulation-gem DA branch?

Reply:
wait, no we don't want to remove simplecov... because it HAS to be a github checkout, since we have never released our fork. (And simplecov has not removed their JSON dependency).